src/templates/base/2019/base_chapter.html

```diff
@@ -212,11 +212,13 @@
 <main id="chapter" class="main">
     <nav aria-label="{{ self.index_nav_title() }}" class="index">
         <div class="index-box floating-card">
-          <h2 class="header">
-            <button class="index-btn" aria-expanded="false" aria-label="{{ self.open_the_index() }}">{{ self.index_title() }}</button>
-            <span class="no-button">{{ self.index_title() }}</span>
-          </h2>
-          {{ self.index() }}
+          <div class="index-scroller">
```

Comment:
Is this extra div needed? Could we add the `index-scroller` class to the `index-box` element?

Reply:
Lost the rounded corners as scrollbar only has box corners.

Reply:
Ok good catch

Reply:
Chrome for example:

![image](https://user-images.githubusercontent.com/10931297/79351499-a9abfc80-7f30-11ea-96d7-39915de246fd.png)

![image](https://user-images.githubusercontent.com/10931297/79351889-1de6a000-7f31-11ea-98bb-4efaaf970489.png)

Though now I look at it again I think the straight edge is better as scrollbar is flush with right hand side. Will change it.